e2e_tests/e2e_test_setup.sh

```diff
@@ -12,7 +12,7 @@ while getopts "CS" o; do
             then
                 echo "Clearing docker compose data"
                 cd nigiri
-                docker-compose down
+                docker compose down
```

Comment:
nit: These changes (if committed on purpose) would be nice in a separate commit. I don't see a relation to the code changes.

Reply:
wasn't on purpose...